components/style/values/specified/list.rs

```diff
@@ -8,6 +8,44 @@ use cssparser::{Parser, Token};
 use parser::{Parse, ParserContext};
 use std::fmt;
 use style_traits::{ParseError, StyleParseErrorKind, ToCss};
+use values::{Either, None_};
+use values::specified::UrlOrNone;
+
+/// Specified and computed `list-style-image` property.
+///
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToCss)]
+pub struct ListStyleImage(pub UrlOrNone);
+
+// FIXME(nox): This is wrong, there are different types for specified
+// and computed URLs in Servo.
+trivial_to_computed_value!(ListStyleImage);
+
+impl ListStyleImage {
+    /// Initial specified value for `list-style-image`.
+    ///
+    #[inline]
+    pub fn get_initial_specified_value() -> ListStyleImage {
```

Comment:
Let's just define `none` here, and remove the `impl` block in the `computed` module, actually.

Reply:
Hey, thanks for reviewing. I'm not sure I understand the `none` definitions. Can you explain it further? (I'm beginning at Rust)

Reply:
Sure! I'm just saying that calling this method `none` (which is the value it returns) is slightly more idiomatic.

That way, you call it like `ListStyleImage::none()`, instead of `get_initial_value`, which is not very useful as a name to know what value is it.